bitbots_ball_filter/cfg/BallFilter.cfg

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+PACKAGE = "bitbots_ball_filter"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+import rospkg
+
+rospack = rospkg.RosPack()
+package_path = rospack.get_path('bitbots_ball_filter')
+
+gen = ParameterGenerator()
+
+group_ROS= gen.add_group("ROS", type="tab")
+group_filter= gen.add_group("Filter", type="tab")
+
+group_ROS.add("ball_subscribe_topic", str_t, 0, "", None)
+group_ROS.add("ball_position_publish_topic", str_t, 0, "", None)
+group_ROS.add("ball_movement_publish_topic", str_t, 0, "", None)
+group_ROS.add("ball_publish_topic", str_t, 0, "", None)
+group_ROS.add("filter_frame", str_t, 0, "", None)
+
+
+group_filter.add("filter_rate", int_t, 0, "Filtering rate in Hz", min=0, max=255)
+group_filter.add("velocity_reduction", double_t, 0, "Velocity reduction (per axis) factor of the ball per second", min=0, max=1.0)
+group_filter.add("process_noise_variance", double_t, 0, "", min=0, max=1.0)
```

Comment:
Please add a parameter desciption

Reply:
And evaluate the value range

Reply:
@NFiedler can you describe, what this param does, please?